src/main/java/com/rackspace/salus/monitor_management/web/controller/MonitorApiController.java

```diff
@@ -330,4 +331,10 @@ public void delete(@PathVariable String tenantId,
   public MultiValueMap<String, String> getMonitorLabelSelectors(@PathVariable String tenantId) {
     return monitorManagement.getTenantMonitorLabelSelectors(tenantId);
   }
+
+  @GetMapping("/tenant/{tenantId}/search/{searchCriteria}")
+  @ApiOperation("Finds all monitors that match the searchCriteria either in the monitorName or the ID")
+  public PagedContent<Monitor> getMonitorLabelSelectors(@PathVariable String tenantId, @PathVariable String searchCriteria, Pageable pageable) {
```

Comment:
Same thing as https://github.com/racker/salus-telemetry-resource-management/pull/82#discussion_r451057125, so I'd suggest:
```suggestion
  @GetMapping("/tenant/{tenantId}/search")
  @ApiOperation("Finds all monitors that match the searchCriteria either in the monitorName or the ID")
  public PagedContent<Monitor> getMonitorLabelSelectors(@PathVariable String tenantId, @QueryParam("q") String searchCriteria, Pageable pageable) {
```